app/modules/eus/enduser_setup.html

```diff
@@ -153,14 +153,17 @@
 			<div id=deviceId></div>
 			<div id=f1>
 				<h3>Connect device to your Wi-Fi</h3>
-				<button id=networks type=button class=utility></button>
-				<div id=dropdown>
-					<span id=arrow>&#x25bc;</span>
-					<select id=aplist name=aplist></select>
-				</div>
-				<input id=ssid type=text autocorrect=off autocapitalize=none placeholder='Wi-Fi Name' />
-				<input id=wifi_password type=text autocorrect=off autocapitalize=none autocomplete=off placeholder=Password />
-				<button id=submit type=button>Save</button>
+				<form action="/setwifi" method="POST">
+					<button id=networks type=button class=utility></button>
+					<div id=dropdown>
+						<span id=arrow>&#x25bc;</span>
+						<select id=aplist name=aplist></select>
+					</div>
+					<input id=ssid name=wifi_ssid type=text autocorrect=off autocapitalize=none placeholder='Wi-Fi Name' />
+					<input name=wifi_password type=text autocorrect=off autocapitalize=none autocomplete=off placeholder=Password />
```

Comment:
I know it's been like that before but this should really be `type=password`.

Reply:
It really should

Reply:
I will also fix this. And we should mention `app/modules/eus/prepare.sh` in the `.md`. I only just found it when I wondered how to prepare the backup `.def` file.

Reply:
I will also have a look to get it checked out under Windows with Unix linens. Unless you beat

Reply:
Don't worry. I don't work on Windows.

Reply:
all done